docs/installing.rst

```diff
@@ -25,32 +25,36 @@ related to containers and use the discussions_ forum instead.
 
 .. note::
 
-    The default installation of ansible-lint package no longer installs any
-    specific version of ansible. You need to either install the desired version
+    The default installation of the ansible-lint package no longer installs any
+    specific version of Ansible. You need to either install the desired version
     of Ansible yourself or mention one of the helper extras:
 
-    * ``core`` - will install latest version of ansible-base 2.10
-    * ``community`` - will install latest version of ansible 2.10 with community collections
+    * ``core`` - will install the latest version of ansible-base 2.10
+    * ``community`` - will install the latest version of ansible 2.10 with community collections
 
 Using Pip
 ---------
 
+While you can use `pip3` to install it, we do recommend using pipx_ as this
```

Comment:
Single quotes generate an `any` role that tries to link to any Sphinx object with this identifier. So either add pip to intersphinx and use a proper role from https://webknjaz.github.io/intersphinx-untangled/pip.pypa.io/ or use double backticks for the inline code role.